src/widget/form/addfriendform.h

```diff
@@ -73,9 +73,9 @@ private slots:
     void retranslateUi();
     void addFriendRequestWidget(const QString& friendAddress, const QString& message);
     void removeFriendRequestWidget(QWidget *friendWidget);
-    void retranslateAcceptButton(QPushButton* acceptButton);
-    void retranslateRejectButton(QPushButton* rejectButton);
-    void deleteFriendRequest(const QString &toxId);
+    static void retranslateAcceptButton(QPushButton* acceptButton);
+    static void retranslateRejectButton(QPushButton* rejectButton);
+    static void deleteFriendRequest(const QString &toxId);
```

Comment:
If you add statics here, there are a LOT of other places where you could have added static :)